scripts/cibuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+vagrant up --provision
+vagrant ssh -c "
+    cd /vagrant
+    echo -e '\nRunning linter...'
+
+    # run linter with console output
+    npm run lint
```

Comment:
Worth noting the `cibuild` script for the climate API project runs the `test` script, and therefore runs tests and linting, if indirectly. Since the scripting required to do so for this project is considerably less involved, I thought it made sense to consolidate.

Also note that provisioning runs `npm install`, so there is no need for a separate script for that.